pcdsdaq/ext_scripts.py

```diff
@@ -27,36 +27,42 @@ def call_script(args, timeout=None, ignore_return_code=False):
         raise
 
 
-def hutch_name():
+def hutch_name(timeout=None):
+    if timeout is None:
+        timeout = 5
```

Comment:
Why that instead of this?
```suggestion
def hutch_name(timeout=5):
```

Reply:
My mistake, brain was stuck in "make the default None if the expected input is mutable" mode, this should instead be as you suggest.